evergreen/eg_app/consumers.py

```diff
@@ -130,7 +134,7 @@ async def receive(self, text_data=None, bytes_data=None):
     @database_sync_to_async
     def get_all_posts(self):
 
-        Post = apps.get_model("eg_app", "Post")
+        # Post = apps.get_model("eg_app", "Post")
```

Comment:
I forgot to call this out earlier, but all these should be outright deleted, not just commented out